### PR TITLE
Fix resize of menuconfig in Python 3 on MacOs

### PR DESCRIPTION
**Pull request by dobairoland**
Closes https://github.com/ulfalizer/Kconfiglib/issues/84

### CODE REVIEW

**Comment by dobairoland** [comment]
@ulfalizer @kumekay Please take a look. I think this is a sufficient fix.

**Comment by kumekay** [comment]
This looks good and works for me just fine

File: menuconfig.py [hunk 1]
```diff
@@ -3128,9 +3128,10 @@ def _getch_compat(win):
     # Uses get_wch() if available (Python 3.3+) and getch() otherwise. Also
     # handles a PDCurses resizing quirk.
 
-    if hasattr(win, "get_wch"):
+    try:
         c = win.get_wch()
-    else:
+        # curses.error is thrown in Python 3 on MacOs
```
Comment (ulfalizer) [inline menuconfig.py]: Could change the message to say

```
# Work around a crash where get_wch() raises curses.error on macOS when
# resizing the terminal by falling back on getch(). This needs more
# investigation. It doesn't happen on all systems.
```

See https://github.com/ulfalizer/Kconfiglib/issues/84#issuecomment-572450946 too. Would be nice to narrow down the configurations that have this issue.

**Comment by ulfalizer** [comment]
Pushed out a release with a fix (same as this one, just with some more comments): 68bcecd

Would be nice to figure out and report the root issue at some point though.

Thanks.

**Pull request closed without merging**